docs/source/intervention_models/therapeutic_zinc/index.rst

```diff
@@ -0,0 +1,229 @@
+
+According to the Cochrane review performed by [Lazzerini-and-Wanzira-2016]_, zinc supplementation may shorten the average duration of diarrhea by around half a day (-11.46 hours; 95% CI: -19.72, -3.19). Notably, the effect appears to be greater in children with signs of malnutrition at (-26.39 hours; 95% CI: -36.54, -16.23).
+
+Given that:
+
+  duration = duration_covered * coverage + duration_uncovered * (1 - coverage)
+
+  MD = duration_covered - duration_uncovered
+
+Then:
+
+  duration_uncovered = duration - (MD * coverage) 
+
+  duration_covered = duration_covered + MD
+
+And:
+
+  remission_rate_uncovered = 1 / (duration_uncovered / 365)
+
+  remission_rate_covered = 1 / (duration_covered / 365)
+
+So:
+
+  RR = remission_rate_covered / remission_rate_uncovered
+
+  PAF = (coverage * RR + (1-coverage) - 1) / (coverage * RR + (1-coverage))
+
+Then, the impact of the intervention can be applied in Vivarium like so:
+
+  remission_rate_i = remission_rate * (1 - PAF) * RR_i
+
+Where,
+
+.. list-table::
+  :header-rows: 1
+
+  * - Parameter
+    - Definition
+    - Value
+    - Note
+  * - remission_rate
+    - Diarrheal diseases remission rate (per person-year)
+    - Defined on the :ref:`diarrheal diseases cause model document <2019_cause_diarrhea>`
+    - 
+  * - coverage
+    - Baseline coverage of therapeutic zinc
+    - Defined in the table above
+    - 
+  * - MD_hours 
+    - Mean difference in diarrhea duration for therapeutic zinc in hours
+    - -11.46 (95% CI: -19.72, -3.19; lognormal distribution of uncertainty)
+    - From [Lazzerini-and-Wanzira-2016]_
+  * - MD
+    - Mean difference in diarrhea duration for therapeutic zinc in years
+    - MD_hours / 24 / 365
+    - 
+  * - duration
+    - Average duration of diarrhea in years
+    - :ref:`diarrheal diseases cause model document <2019_cause_diarrhea>`
+    - Will need to be converted to years (defined in days)
+  * - RR_i
+    - Diarrheal disease remission rate relative risk for an individual simulant dependant on their therapeutic zinc intervention coverage
+    - RR as calculated above if covered by therapeutic zinc intervention; otherwise 1
+    - 
+
+Assumptions and Limitations
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+#. The average duration of a diarrhea episode used in GBD is not estimated at the location-specific level.
+
+#. We do not consider effect modification by baseline burden of zinc deficiency, level of zinc intake, or zinc fortification coverage. Notably, the PAF for zinc deficiency and diarrheal diseases in Ethiopia as estimated by GBD 2019 among children under five years of age is quite low. However, challenges around the definition and measurement of zinc deficiency may explain this finding. We chose to rather model the directly measured effect of zinc supplementation on diarrheal disease incidence, for which there may be an effect even among those who do not meet the criteria for zinc deficiency.
```

Comment:
Do we want any sort of way to build in an upper bound for the effect zinc could have on diarrhea incidence / remission, given this?

Reply:
I think we'll keep it as is for now given that the impact of the intervention is expected to be really small even under optimistic assumptions and then I will go back and add more detail about zinc deficiency burden in Ethiopia as well as the limitations of the zinc deficiency measurement in GBD